src/Shikimori/composer.lock

```diff
@@ -0,0 +1,3066 @@
+{
```

Comment:
composer.lock should not have been commited.
Can you remove it @M9snikFeed ?

Reply:
Yes

Reply:
You didn't remove it